app/scripts/lib/auth-errors.js

```diff
@@ -64,10 +67,12 @@ define(function (require, exports, module) {
     },
     INVALID_PARAMETER: {
       errno: 107,
+      errorPageBaseUrl: BAD_REQUEST_PAGE,
```

Comment:
Instead of a separate list, why not make the error page part of the error configuration? Does `errorPageBaseUrl` make sense? Too verbose? Not descriptive enough? Too descriptive?